docs/trestle_author.md

```diff
@@ -18,347 +27,573 @@ While trestle provides editing support for OSCAL there is an unfortunate truth t
+         **Technical Approver:**   John Doe (approved)
+         **Version:** 1.0.1
+      ```
+      Running `trestle author docs validate -tn docs_task -gh="Governed section"` will ensure this content is present in the document.
+    - If `--ignore ^_.*` (`-ig`) is passed it will validate all files except folders and files that start with underscore `_`. Use this option when you would like to ignore any folders or files that match given regular expression.
+    - If `--header-validate` (`-hv`) is passed the header will be validated as well.
+    - If `--header-only-validate` (`-hov`) is passed only the header and NOT the body will be validated.
+    - If `--readme-validate` (`-rv`) is passed README.md will be validated as well, otherwise it is ignored.
+    - If `--recurse` (`-r`) is passed the documents in the subfolders will also be validated. By default `author docs` only indexes a flat directory.
+    - If `--template-version 1.0.0` (`-tv`) is passed the header field `x-trestle-template-version` will be ignored and document will be forcefully validated against template of version `1.0.0`.
+      Use this for testing purposes _only_ when you need to validate the document against a specific template. By default the template version will be determined based on `x-trestle-template-version` in the document.
 
-- For a template, if the x-trestle-template-version header exists, then it and the Version header of that template must
-  always match. Further, the template's path needs to include the value stored in the x-trestle-template-version.
-- For an instance, the Version header should not be compared with the Version header of the template. Rather,
-  the x-trestle-template-version header should be compared with the x-trestle-template-version header of the template.
+    </details>
 
-## `trestle author folders`
+  - <details>
+      <summary><b>folders - Validating documents against multiple templates and ensuring folder structure (multiple individual templates per task) </b></summary>
 
-`author folders` is designed to allow the assembly of groups of templates where the folder assembly is the unique instance. Trestle author folders supports validation of both markdown and drawio files. Note that headers / metadata must be specified in each applicable template.
+    ## Validating against multiple templates
 
-For example given the following template setup using `trestle author folders setup -tn my_task_2`
+    In this section we describe the functionality of `trestle author folders` command.
 
-```text
-trestle_root
- ┣ .trestle
- ┃ ┣ author
- ┃ ┃ ┗ my_task_2
- ┃ ┃ ┃ ┣ a_template.md
- ┃ ┃ ┃ ┣ another_template.md
- ┃ ┃ ┃ ┗ template.drawio
- ┃ ┗ config.ini
-```
+    `author folders` is designed to allow the assembly of groups of templates where the folder assembly is the unique instance. Trestle author folders supports validation of both markdown and drawio files. Note that headers / metadata must be specified in each applicable template.
```

Comment:
Please expand on what you mean by "unique instance".  I think a few sentences are needed.

Reply:
Done